internal/static/i18n/de.yaml

```diff
@@ -715,6 +723,10 @@ EventTypes:
         added: Datenschutzbestimmung und AGB hinzugefügt
         changed: Datenschutzbestimmung und AGB geändert
         removed: Datenschutzbestimmung und AGB entfernt
+      notification:
+        added: Notifikation Richtlinie hinzugefügt
+        changed: Notifikation Richtlinie geändert
+        removed: Notifikation Richtlinie entfernt
     flow:
```

Comment:
that's because the errors(Errors.IAM.NotificationPolicy.NotChanged) and eventtypes(policy.notification.added) are differently structured, as we anyway have to look at our translation-management sometime soon, maybe we should pick it up then again